org-formation/organization.yaml

```diff
@@ -563,6 +564,22 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         <<: !Include ./_platform_org_tags.yaml
 
+  RecoverProdAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-recover-prod
+      AccountId: '634761300905'
```

Comment:
Omit this line (`AccountId: '634761300905'`)?

Reply:
Done